src/components/elementView/elementTab.tsx

```diff
@@ -241,9 +228,9 @@ class ElementTab extends React.Component<ElementTabProps, ElementViewState> {
         ? "api/sites/" + path + "/get-many"
         : "api/" + path + "/get-many";
     return axios.post(API_ROOT + url, bodyCopy, config).then((res) => {
-      const items = res.data[path];
-
-      return items;
+      const dataKey =
+        path === "offline-storage-sites" ? "offline_storage_sites" : path;
+      return res.data[dataKey];
```

Comment:
everything else is one word so this wasn't a problem before offline storage sites